### ISSUE TITLE
Channels are marked as read when navigating away from the buffer

### ISSUE THREAD

**Issue by haldean** [wee-slack/wee-slack#319]
Steps to repro:

1. Open channel `#a` in weechat
2. Open channel `#a` on the slack website
3. Have a friend send a message to `#b`
4. Switch to `#b` in weechat

After 4, the channel still appears as unread on the slack website, even though (IMO) it should be marked as read, since I switched to it in weechat. If, in weechat, I switch back from `#b` to `#a`, then `#b` is correctly marked as read in the web client.

Not sure if this is a limitation of writing a weechat plugin or just a bug, but I thought I'd mention it either way. Thanks so much!

**Comment by trygveaa** [comment]
This is actually the intended behavior. The reasoning is that you haven't read the messages immediately, and the read marker in weechat is still above the new messages. With the newest version in master (after #441), if you run `/input set_unread_current_buffer` to move the read marker to the bottom, the channel will also be marked as read on Slack.

**Comment by haldean** [comment]
I guess, from my perspective, navigating to a channel is a better indication that the message has been read then navigating away from the same channel (which is the current behavior). That said, I definitely understand the reasoning! Thanks for the response.

**Comment by mtfurlan** [comment]
What about after a user sends a message?
I think that at that point, wee-slack should mark the channel as read.

**Comment by erdnaxeli** [comment]
I totally agree with @mtfurlan, but it worth noticing than weechat _does not_ remove the read marker when sending a message on a channel neither.
